tests/tests_suite_test.go

```diff
@@ -76,7 +76,6 @@ func TestTests(t *testing.T) {
 	suiteConfig, _ := GinkgoConfiguration()
 	if suiteConfig.ParallelTotal > 1 {
```

Comment:
I think serial failures will collide with parallel ones
(https://github.com/kubevirt/kubevirt/pull/8770/commits/f5c91d99b8597e39f86e491088c46a07ec26334b)

Reply:
ping @dhiller

Reply:
No they won't. Ginkgo runs tests with [`Serial` decorator](https://onsi.github.io/ginkgo/#serial-specs) on one of the test nodes after all the parallel tests have been run.

Reply:
I think it's possible to have a parallel test `i` fail in node 1's parallel run, and then have test `i` fail in the serial run as well, overwriting each other's artifacts